bot/cogs/filtering.py

```diff
@@ -52,8 +57,12 @@ def expand_spoilers(text: str) -> str:
 class Filtering(Cog):
     """Filtering out invites, blacklisting domains, and warning us of certain regular expressions."""
 
+    # Redis cache for last bad words in nickname alert sent per user.
```

Comment:
Comment could probably be a little more descriptive:
```suggestion
    # Redis cache mapping a user ID to the last timestamp a bad nickname alert was sent
```